src/components/navbar/Wallet.tsx

```diff
@@ -79,7 +79,9 @@ const AccountContainer = ({ account }: { account?: string }) => {
   return (
     <div
       className={accountLinkContainer}
-      onClick={async () => await WalletStore.connect(true)}
+      onClick={async () => {
+        await WalletStore.connectOrChangeNetwork(true, needNetworkChange)
```

Comment:
Can we make the arguments as object, so we can understand how we connect the wallet without going into the store to check what these arguments mean?